src/open_inwoner/plans/tests/test_logging.py

```diff
@@ -32,9 +32,9 @@ def setUp(self):
 
     def test_created_plan_is_logged(self):
         plan = PlanFactory.build(created_by=self.user)
-        form = self.app.get(reverse("collaborate:plan_create"), user=self.user).forms[
-            "plan-form"
-        ]
+        form = self.app.get(
+            reverse("collaborate:plan_create_no_template"), user=self.user
```

Comment:
```suggestion
            reverse("collaborate:plan_create"), user=self.user
```
I would prefer keeping `plan_create` here and elsewhere for creation of plans without template, though perhaps @swrichards has a different view of the naming here.

Reply:
I wanted to give a very specific, more explicit meaning to the new situation where there is no such thing as just 'creating' a plan. 
Also we are sort of trying to hide the original 'create' default, and we wish to push the creating-plan-from-template situation.

I can of course reverse all of the `plan_create_no_template` to `plan_create` again if we all understand what it is for 😇 - just let me know.